src/global_styling/utility/utility.tsx

```diff
@@ -9,4 +9,14 @@
 import React from 'react';
 import { Global, css } from '@emotion/react';
 
-export const EuiUtilityClasses = () => <Global styles={css``} />;
+import { euiScreenReaderOnly } from '../../components/accessibility/screen_reader_only/screen_reader_only.styles';
+
+export const EuiUtilityClasses = () => (
+  <Global
+    styles={css`
+      .euiScreenReaderOnly {
+        ${euiScreenReaderOnly()}
+      }
+    `}
+  />
+);
```

Comment:
Optional suggestion, but it might be easier to read the css styles as its own top level variable rather than as a nested prop:

```suggestion
const globalStyles = css`
.euiScreenReaderOnly {
  ${euiScreenReaderOnly()}
}
`;

export const EuiUtilityClasses = () => (
  <Global styles={globalStyles} />
);
```